lib/views/frontend/spree/checkout/payment/_stripe.html.erb

```diff
@@ -1,66 +1,67 @@
-<%= render "spree/checkout/payment/gateway", payment_method: payment_method %>
```

Comment:
I'm inlining the gateway partial here to make the changes for the Stripe upgrade.